Makefile.in

```diff
@@ -27,9 +27,12 @@ endif
 
 ifeq ($(shell expr $(KERNELVER) \>= $(FPATH_MINKVER)),1)
 PXDEFINES:=$(PXDEFINES) -D__PX_FASTPATH__
```

Comment:
This can't be a compile time option. It needs to be a configurable, we need a way out if something goes wrong.

Reply:
https://github.com/portworx/px-fuse/pull/59 addresses this.